rosie.py

```diff
@@ -31,7 +30,12 @@ def run():
         exit(1)
     target_directory = argv[3] if len(argv) >= 4 else '/tmp/serenata-data/'
```

Comment:
Just find out the problem here: if we update the value, we need to check here: `if len(argv) >= 4 else '/tmp/serenata-data/'`

Reply:
What I plan to do, like @Irio's tag `--years`, I'll add a tag for path `--path` so if there is an path, it'll be the next argument